config/jobs/periodic/cluster-api-provider-ibmcloud/test-e2e-capi-ibmcloud-periodics.yaml

```diff
@@ -60,8 +26,5 @@ periodics:
           command:
             - "runner.sh"
             - "./scripts/ci-e2e.sh"
-          envFrom:
```

Comment:
why removed this section? not used anymore, adding some description is always helpful to the PR.

Reply:
@Amulyam24 mentioned that we are updating the API key via boskos.
for more context, the env var will be fetched here - https://github.com/Amulyam24/cluster-api-provider-ibmcloud/blob/ee12c916c92ab3aa0cc5be6f4a52402ea1ac5d0c/hack/boskos.sh#L44